lib/src/textboxGroup/textboxGroup.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="form-group input-box-group" [ngClass]="className">
+    <ng-container *ngIf="label">
+        <label class="custom-label" [htmlFor]="internalId">
+            {{ label }}
+        </label>
+    </ng-container>
+    <div class="input-group" [ngClass]="{ success: success, 'has-error': error }" [class.disabled]="disabled">
```

Comment:
```suggestion
    <div class="input-group" [ngClass]="{ success: success, 'has-error': error, disabled: disabled }">
```

Reply:
I recognised and agree with above @omarVodiaK . But i refused to combine them because i feel like while there is a relationship between success and error, i feel like disabled is separated and wanted to preserve that distinction. This however is just a preference only. I will revert it back if it has any bearing on the feature or code quality.

Reply:
they are all classNames.

Reply:
Ok the disabled kid is placed in the same class.